src/main/resources/data/bloodmagic/patchouli_books/guide/ru_ru/templates/crafting_soulforge.json

```diff
@@ -0,0 +1,84 @@
+{
+  "processor": "wayoftime.bloodmagic.compat.patchouli.processors.TartaricForgeProcessor",
+  "components": [
+    {
+      "type": "header",
+      "text": "#heading",
+      "x": -1,
+      "y": -6
+    },
+    {
+      "type": "image",
+      "image": "bloodmagic:textures/gui/patchouli_book/crafting.png",
+      "x": 11,
+      "y": 4,
+      "u": 0,
+      "v": 111,
+      "texture_width": 128,
+      "texture_height": 256,
+      "width": 93,
+      "height": 43
+    },
+    {
+      "type": "item",
+      "item": "#input1",
+      "x": 15,
+      "y": 8
+    },
+    {
+      "type": "item",
+      "item": "#input2",
+      "x": 34,
+      "y": 8
+    },
+    {
+      "type": "item",
+      "item": "#input3",
+      "x": 15,
+      "y": 27
+    },
+    {
+      "type": "item",
+      "item": "#input4",
+      "x": 34,
+      "y": 27
+    },
+    {
+      "type": "item",
+      "item": "#will",
+      "x": 59,
+      "y": 27
+    },
+    {
+      "type": "item",
+      "item": "#output",
+      "x": 84,
+      "y": 8
+    },
+	{
+      "type": "item",
+      "item": "bloodmagic:soulforge",
+      "x": 84,
+      "y": 27
+    },
+    {
+      "type": "tooltip",
+      "tooltip": ["Minimum: #willrequired# Will", "Drained: #willdrain# Will"],
+      "x": 59,
+      "y": 8,
+      "width": 16,
+      "height": 16
+    },
+    {
+      "type": "text",
+      "text": "Will",

```

Comment:
Text over the recipe's arrow to indicate the tooltip exists.  Need to translate "Will" [Demon Will].